modules/v2/main.tf

```diff
@@ -32,6 +32,10 @@ locals {
     email  = google_service_account.sa[0].email,
     member = google_service_account.sa[0].member
   } : {}
+  apphub_service_uri = {
+    service_uri = "//run.googleapis.com/projects/${var.project_id}/locations/${var.location}/services/${var.service_name}"
```

Comment:
Lets interpolate the computed id attrib rather than the input variable var.service name (both and here and the other module). Since only used in a single place, we don't need to define it as a local, we can directly have this in outputs.tf
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#id

```
output "apphub_service_uri" {
  value       = {
  service_uri = ...
}
  description = "Service URI in CAIS style to be used by Apphub."
}
```

Reply:
TIL. Thanks for pointing out the computed attribute!